src/Sylius/Bundle/ResourceBundle/EventListener/ORMTranslatableListener.php

```diff
@@ -214,4 +212,9 @@ private function hasUniqueConstraint(ClassMetadata $metadata, array $columns): b
 
         return false;
     }
+
+    private function getTranslatableEntityLocaleAssigner(): TranslatableEntityLocaleAssignerInterface
+    {
+        return $this->container->get('sylius.translatable_entity_locale_assigner');
```

Comment:
Is it really required to inject container rather than specific service?

Reply:
It has been already injected since pre-stable, so it is covered by our BC policy. I'll create an issue about how to deprecate that though.

Reply:
https://github.com/Sylius/Sylius/issues/9677

Reply:
In the end, it was a fault of missing ProxyManagerBridge, so I reverted the changes made previously in this file, but the issue is still relevant.